internal/providers/azure/azure.go

```diff
@@ -53,49 +47,43 @@ const (
 	CDS_DISC_OK
 )
 
-// These constants are the types of CDROM filesystems that might
-// be used to present a custom-data volume. Azure proper uses a
-// udf volume, while Azure Stack might use udf or iso9660.
-const (
-	CDS_FSTYPE_UDF     = "udf"
-	CDS_FSTYPE_ISO9660 = "iso9660"
-)
-
+// FetchConfig is a wrapper around GetFetchConfig.
 func FetchConfig(f *resource.Fetcher) (types.Config, report.Report, error) {
-	devicePath := filepath.Join(distro.DiskByIDDir(), configDeviceID)
+	return GetFetchConfig(f, NewAzureConfig([]string{CONFIG_DVD_FS_UDF}))
+}
 
+// GetFetchConfig implements the platform.NewFetcher interface.
```

Comment:
It doesn't, because of the extra argument.  This should move back to `FetchConfig`.

Also, weird name :slightly_smiling_face: